srcCs/RTIDDSImpl.cs

```diff
@@ -996,9 +996,9 @@ public long getPulledSampleCount()
                 return status.pulled_sample_count;
             }
 
-            public void wait_for_acknowledgments(Duration_t timeout){
+            public void wait_for_acknowledgments(int sec, uint nanosec){
```

Comment:
Could you add a space between ")" and "{"

Reply:
Done